syntax = "proto2";
message InterliveServerStat
{
    optional uint64    sys_uptime                       = 1;    // Seconds since system boot. collect:0|10|
    optional int32     sys_cpu_cores                    = 2;    // CPU cores count. collect:0|
    optional int32     sys_cpu_idle                     = 3;    // Percentage of current system CPU IDLE. collect:10|
    optional int32     sys_mem_total                    = 4;    // Total physical memory of system in KB. collect:0|
    optional int32     sys_mem_total_free               = 5;    // Total availible physical memory of current system in KB. Equals to MemFree+Buffers+Cached. collect:0|10|
    optional uint64    sys_net_rx                       = 6;    // Total received Bytes of the first public network. collect:10|
    optional uint64    sys_net_tx                       = 7;    // Total sent Bytes of the first public network. collect:10|
    optional int32     proc_cpu_use                     = 8;    // Percentage of current process CPU use. collect:10|
    optional int32     proc_sleepavg                    = 9;    // Percentage of current process CPU sleep average. collect:10|
    optional int32     proc_vmpeak                      = 10;   // Peak alloced physical memory of process in KB. collect:10|
    optional int32     proc_vmsize                      = 11;   // Total used physical memory of process in KB. collect:10|
    optional uint64    proc_total_uptime                = 12;   // Process running time in Seconds. collect:10|
    optional string    proc_process_name                = 13;   // Process name. collect:0|
    optional int32     buss_fsv2_stream_count           = 14;   // forward server v2 current stream count. collect:10|
    optional int32     buss_fsv2_total_session          = 15;   // forward server v2 session count. collect:10|
    optional int32     buss_fsv2_active_session         = 16;   // forward server v2 active session count. collect:10|
    optional int32     buss_fsv2_connection_count       = 17;   // forward server v2 current connection count. collect:10|
    optional int32     buss_fcv2_stream_count           = 18;   // forward client v2 current stream count. collect:10|
    optional int32     buss_uploader_connection_count   = 19;   // Current uploading stream count. collect:10|
    optional int32     buss_player_connection_count     = 20;   // Current downloading stream count. collect:10|
    optional int32     buss_player_online_cnt           = 21;   // Current player online  count. collect:10|
    optional int32     proc_vmrss                       = 22;   // Really used physical memory of process in KB. collect:10|
}
